plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeCommitSummary.java

```diff
@@ -26,16 +28,22 @@
 
 public class DeltaLakeCommitSummary
 {
+    private final long version;
     private final List<MetadataEntry> metadataUpdates;
     private final Optional<ProtocolEntry> protocol;
+    private final Optional<CommitInfoEntry> commitInfo;
+    private final List<RemoveFileEntry> removedFiles;
```

Comment:
Could you summarize the data in this summary more? Looks like you could throw out any add file commits that are blind appends entirely, and for other commits you need a set of the partitions but not the rest of the data.

Reply:
Keeping only:
- canonical partition keys for the added files
- a flag stating whether or not the commit contains remove files.